packages/api/src/models/webhook.ts

```diff
@@ -28,6 +33,22 @@ const WebhookSchema = new Schema<IWebhook>(
       required: true,
     },
     url: {
+      type: String,
+      required: false, // TODO: should this not be required?
+    },
+    description: {
+      type: String,
+      required: false,
+    },
+    queryParams: {
+      type: String,
+      required: false,
+    },
+    headers: {
+      type: String,
+      required: false,
+    },
```

Comment:
I wonder if it makes more sense to use `Map` type for headers (https://mongoosejs.com/docs/schematypes.html#maps)

Reply:
ahhh you are very correct, no need to ever stringify the header data, doing a lot of unneeded back and forth there for no reason

Reply:
honestly, they all could be, there is a lot of back and forth parsing it might be better to store everything as k/v pairs hmmm